src/core/server/integration_tests/elasticsearch/capabilities_serverless.test.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import {
+  TestServerlessESUtils,
+  createTestServerlessInstances,
+} from '@kbn/core-test-helpers-kbn-server';
+import type { ElasticsearchClient } from '@kbn/core-elasticsearch-server';
+import { getCapabilitiesFromClient } from '@kbn/core-elasticsearch-server-internal';
+
+// skipped because test serverless ES nodes are currently using static ports
+// causing parallel jest runners to fail for obvious port conflicts reasons.
+describe.skip('ES capabilities for serverless ES', () => {
```

Comment:
See comment. The test passes locally, but I didn't want to introduce more flakiness right now, so I'm commenting it until the serverless test utils become more mature.

Reply:
FWIW I am working on adding similar tests for all migration actions here: https://github.com/elastic/kibana/pull/164842

Still good to have the capabilities covered more specifically.